config/environments/production.rb

```diff
@@ -16,10 +16,9 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
```

Comment:
Was this change generated or did you change it yourself?

Reply:
This was generated from the app:update task. I believe it's because rails 5.2 refactored the secrets handling. In my own applications, we don't use rails' mechanism for secrets, though, and I figured that might also be true for you guys. If that's wrong, though, we should be careful here. If there's a staging environment we could test it there first.

Reply:
By 
> rails' mechanism for secrets

 do you mean the `/config/secrets.yml`?

Reply:
That's correct, nor do we make use of the secrets-related config attributes, which are what changed in this initializer

Reply:
Based on the comment, it seems that if you are setting `ENV["RAILS_MASTER_KEY"]`, we can enable this setting and things will continue to work. But if you don't use that and instead have the master key in `config/secrets.yml.key` it looks like on rails 5.2 it'll need to be in `config/master.key`.